src/server/migrations/20190918172031-scenarios.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+exports.up = function(db) {
+    return db.runSql(
+        `
+CREATE OR REPLACE FUNCTION updated_at() RETURNS TRIGGER AS $$
```

Comment:
if we split all the tables up - i might suggest we split this up into it's own "functions" too